cmd/token.go

```diff
@@ -101,8 +104,18 @@ The OIDC token includes claims regarding the GCE VM, which is verified by Attest
 		}
 		gceAK.Close()
 
+		cloudLogEnabled, _ := cmd.Flags().GetBool("cloudLog")
+		if cloudLogEnabled {
+			cloudLogClient, err = logging.NewClient(ctx, projectID)
+			if err != nil {
+				return fmt.Errorf("failed to create cloud logging client: %w", err)
+			}
+			cloudLogger = cloudLogClient.Logger("go-tpm-tool")
```

Comment:
`gotpm` is the current name of the CLI tool. Please extract a const variable for this name in case we change it.